contentcuration/contentcuration/api.py

```diff
@@ -15,13 +15,7 @@
 import models
 
 def count_files(node):
-    count = 0
-    if node.kind_id == "topic":
-        for n in node.children.all():
-            count += count_files(n)
-    else:
-        count += models.File.objects.filter(contentnode=node).count()
-    return count
+    return node.get_descendants().exclude(kind_id="topic").count()
```

Comment:
depends on what files you want to count, there are cases where a contentnode can have more than one file, such as video and caption.